code/modules/surgery/wound_repair.dm

```diff
@@ -62,7 +64,7 @@
 			user.visible_message(SPAN_NOTICE("[user] treats the brute damage to [target]'s body with the [tool_name]."), \
 			SPAN_NOTICE("You treat the brute damage to [target]'s body with [tool_name].") )
 			if(tool.use(1))
-				target.adjustBruteLoss(-15)
+				target.adjustBruteLoss(-30)
```

Comment:
Same question, why are you doubling the heal from surgery without mentioning it in the PR body ?

Reply:
I copied the code for the advanced burn and trauma kits as these are effectively reskins of those items.  If the values are some how not the same as those i am unsure how that even happened.

Reply:
This bit of code has nothing to do with ATKs and ABKs, it's the code for the wound repair surgery. It couldn't have magically changed if you didn't go in the file to modify the values.
Once again, here is current master-branch code.

![image](https://user-images.githubusercontent.com/65850818/94663661-347b8a80-030a-11eb-8f4e-67e2877a8636.png)